packages/server/lib/gui/events.coffee

```diff
@@ -207,6 +208,7 @@ handleEvent = (options, bus, event, id, type, arg) ->
         bus.emit("project:error", errors.clone(err, {html: true}))
 
       onWarning = (warning) ->
+        warning.message = stripAnsi(warning.message)
```

Comment:
this shouldn't be necessary

Reply:
besides the message is also in the first line of the stack trace... something feels off about this